frontend/playwright/ui/specs/workspace.spec.js

```diff
@@ -67,6 +67,31 @@ test("User adds a library and its automatically selected in the color palette",
   await expect(workspacePage.palette.getByText('There are no color styles in your library yet')).toBeVisible();
 });
 
+test("BUG 7760 - Layout losing properties when changing parents", async ({ page }) => {
+  const workspacePage = new WorkspacePage(page);
+  await workspacePage.setupEmptyFile();
+  await workspacePage.mockRPC(/get\-file\?/, "workspace/get-file-7760.json");
+  await workspacePage.mockRPC("get-file-fragment?file-id=*&fragment-id=*", "workspace/get-file-fragment-7760.json");
+  await workspacePage.mockRPC("update-file?id=*", "workspace/update-file-create-rect.json");
+
+  await workspacePage.goToWorkspace({fileId: "cd90e028-326a-80b4-8004-7cdec16ffad5", pageId: "cd90e028-326a-80b4-8004-7cdec16ffad6"});
+
+  // Select the flex board and drag it into the other container board
+  await workspacePage.clickLeafLayer("Flex Board");
+
+  // Move the first board into the second
+  const hAuto = await workspacePage.page.locator(`[for='behaviour-h-auto']`);
+  const vAuto = await workspacePage.page.locator(`[for='behaviour-v-auto']`);
+
+  await expect(hAuto).toHaveClass(/checked/);
+  await expect(vAuto).toHaveClass(/checked/);
```

Comment:
We should avoid checking by class whenever possible, since it's very tied to the implementation.

Here we have an alternative, which is to check whether the checkbox is checked.

Ideally we should be able to use `getByRole("checkbox")`, but since it's hidden with `display: none` (it shouldn't be hidden that way, we should eventually change this) it does not appear in the accesibility tree and we cannot use role queries.

In the meantime, we can do:

```js
 await expect(vAuto.locator("input")).toBeChecked();
```

(maybe let's put a `TODO` comment so we don't forget to change this once we have these radio buttons coded properly in the DS)

Reply:
This won't work. I've checked and the radio buttons inside won't update the checked property.